README.md

```diff
@@ -133,7 +133,7 @@ Feel free to reach out if you have any questions. The maintainers of this projec
 DingTalk:
 
 <div>
-  <img src="https://fluid-imgs.oss-cn-shanghai.aliyuncs.com/public/imgs/dingding_code.png" width="280" title="dingtalk">
```

Comment:
Could you also please update [README-zh_CN.md](https://github.com/fluid-cloudnative/fluid/blob/master/README-zh_CN.md).

Reply:
they all refer the same picture. So [README-zh_CN.md](https://github.com/fluid-cloudnative/fluid/blob/master/README-zh_CN.md) has been update too.

Reply:
Thanks， I thought [README-zh_CN.md](https://github.com/fluid-cloudnative/fluid/blob/master/README-zh_CN.md) also referred  https://fluid-imgs.oss-cn-shanghai.aliyuncs.com/public/imgs/dingding_code.png.